batching-gson/src/main/java/com/flipkart/batching/gson/adapters/KnownTypeAdapters.java

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
Added this class since it is published in the internal artifactory, will remove this once the stag PR gets merged